daemons/gptp/common/ptp_message.cpp

```diff
@@ -1467,6 +1469,8 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 		 remote_req_rx_timestamp.nanoseconds);
 
 	// Adjust turn-around time for peer to local clock rate difference
+	// TODO: Are these .998 and 1.002 specifically defined in the standard?
```

Comment:
My vote is that if this is not resolved today, this should not block pull request - we will create a gPTP github issue to review this at a later date.